k8s/namespaces/kube-system/nodelocaldns/patches/sandbox/nodelocaldns.yaml

```diff
@@ -0,0 +1,6 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: node-cache
+          image: lbitonti/k8s-dns-node-cache-amd64:1.15.16-8-g5121975
```

Comment:
Why not using upstream?

Reply:
https://github.com/kubernetes/dns/pull/367